docs/how-to-guides/adding-or-reusing-tests.md

```diff
@@ -161,6 +161,13 @@ The key fixtures are the `environment` and `universal_data_sources` fixtures, wh
 * In `repo_configuration.py` add a new`IntegrationTestRepoConfig` or two (depending on how many online stores you want to test).
 * Run the full test suite with `make test-python-integration.`
 
+### Including a new offline / online store in the main Feast repo as part of the contributions.
```

Comment:
"as part of the contributions" seems weird?

Reply:
Fixed.